app/models/concerns/hyrax/with_events.rb

```diff
@@ -5,7 +5,7 @@ def stream
     end
 
     def event_class
-      self.class.name
+      model_name.name
```

Comment:
I'm wondering about an alias of `event_class` and then deprecation of `event_class`. When I see a method `*_class` my assumption is that I have a `Class` object. However, in reading this, that is not the case. We have a String.

Reply:
yeah. i expect there's more work to do here. `event_namespace` or `event_type` would make more sense.

but i'm also trying to take things step by step and just understand what's really happening here. these events are (almost?) exclusively published in jobs which have a hard-to-follow inheritance structure.

this one change seemed important to me because i'm worried about tight class name dependencies in the AF->Valkyrie transition.

also, it was end-of-day friday, so i submitted the scope of what i actually had done/ready. :)

Reply:
Yeah. I was looking briefly at this, and its a Mixin that would benefit with a class to encapsulate the public behavior. However, as it stands, these PRs move the shared understanding forward, so "Make it so!"